app/views/devise/passwords/edit.html.erb

```diff
@@ -0,0 +1,30 @@
+<% content_for(:title, 'Settings') %>
```

Comment:
Convert to haml or slim. Here and in other views.
See https://github.com/heartcombo/devise/wiki/How-To:-Create-Haml-and-Slim-Views